Sources/Private/Experimental/Animations/RectangleAnimation.swift

```diff
@@ -0,0 +1,153 @@
+// Created by Cal Stephens on 12/21/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import QuartzCore
+
+extension CALayer {
+  /// Adds animations for the given `Rectangle` to this `CALayer`
+  func addAnimations(
+    for rectangle: Rectangle,
+    context: LayerAnimationContext)
+  {
+    addAnimation(
+      for: .path,
+      keyframes: rectangle.size.keyframes,
+      value: { sizeKeyframe in
+        let size = sizeKeyframe.sizeValue * 0.5
```

Comment:
Pulled this `CGPath` rendering implementation from https://github.com/airbnb/lottie-ios/blob/master/Sources/Private/NodeRenderSystem/Nodes/PathNodes/RectNode.swift#L83

Reply:
Do you think it makes sense to try to keep these in sync/shared? Or at least add a comment to the effect that we're trying to keep them in sync? Not sure how long you'd imagine we'll have both

Reply:
Good idea, seems reasonable to keep this shared between the two since it's rather large and unlikely to change much